core/src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -1020,8 +1020,11 @@ trait OptimisticTransactionImpl extends TransactionalWrite
     allowFallbackToSnapshotIsolation
   }
 
-  protected def getDefaultIsolationLevel(): IsolationLevel = {
-    Serializable
+  /**
+   * Default [[IsolationLevel]] as set in table metadata.
+   */
+  private[delta] def getDefaultIsolationLevel(): IsolationLevel = {
```

Comment:
Can we rename it to `defaultIsolationLevel` (to make it less-Java and slightly more Scala)?